src/backend/distributed/metadata/metadata_cache.c

```diff
@@ -191,9 +191,9 @@ static ScanKeyData DistObjectScanKey[3];
 /* local function forward declarations */
 static bool IsCitusTableViaCatalog(Oid relationId);
 static ShardCacheEntry * LookupShardCacheEntry(int64 shardId);
-static DistTableCacheEntry * LookupDistTableCacheEntry(Oid relationId);
-static void BuildDistTableCacheEntry(DistTableCacheEntry *cacheEntry);
-static void BuildCachedShardList(DistTableCacheEntry *cacheEntry);
+static CitusTableCacheEntry * LookupLookupCitusTableCacheEntry(Oid relationId);
```

Comment:
I missed this. Maybe `GetCitusTableCacheEntry` would be preferable for the public-facing API?